src/PrizePool.sol

```diff
@@ -57,12 +54,45 @@ error NoCompletedDraw();
 /// @notice Emitted when a tier does not exist
 error InvalidTier(uint8 tier, uint8 numberOfTiers);
 
+/// @notice Emitted when the sender is not the draw manager
+error SenderIsNotDrawManager(address sender, address drawManager);
```

Comment:
Personal preference but I prefer to use `caller` instead of `sender`:
```suggestion
/// @notice Emitted when the caller is not the draw manager
error CallerNotDrawManager(address caller, address drawManager);
```
We can also add Natspec docs for the parameters.